test/sslapitest.c

```diff
@@ -918,6 +920,114 @@ static int execute_test_ktls(int cis_ktls_tx, int cis_ktls_rx,
     return testresult;
 }
 
+#define SENDFILE_SZ                     (16 * 4096)
+#define SENDFILE_CHUNK                  (4 * 4096)
+#define min(a,b)                        ((a) > (b) ? (b) : (a))
+static int execute_test_ktls_sendfile(void)
+{
+    SSL_CTX *cctx = NULL, *sctx = NULL;
+    SSL *clientssl = NULL, *serverssl = NULL;
+    unsigned char *buf, *buf_dst;
+    BIO *out = NULL, *in = NULL;
+    int cfd, sfd, ffd, err;
+    ssize_t chunk_size = 0;
+    off_t chunk_off = 0;
+    int testresult = 0;
+    FILE *ffdp;
+
+    buf = OPENSSL_zalloc(SENDFILE_SZ);
+    buf_dst = OPENSSL_zalloc(SENDFILE_SZ);
+    if (!TEST_ptr(buf) || !TEST_ptr(buf_dst))
+        goto end;
+
+    if (!TEST_true(create_test_sockets(&cfd, &sfd)))
```

Comment:
We'd typically include this as part of the if above:
```
if (!TEST_ptr(buf)
    || !TEST_ptr(buf_dst)
    || !TEST_true(create_test_sockets(&cfd, &sfd)))
    goto end;
```

Not essential.

Reply:
Done